app/controllers/clients/coordinated_entry_assessments_controller.rb

```diff
@@ -63,8 +64,12 @@ def update
             user_id: current_user.id,
           ),
         )
-        # mark any other actives as inactive
-        @client.ce_assessments.where(active: true).where.not(id: @assessment.id).update_all(active: false)
+        if @assessment.valid?
+          # mark any other actives as inactive
+          @client.ce_assessments.where(active: true).where.not(id: @assessment.id).update_all(active: false)
```

Comment:
Let's move this onto the model `@assessment.make_active!` or some such.  I'm not a fan of these things in a controller that manipulate other instances.

Reply:
Done